File: game_server/ServerProtocol/src/CommonEnum/ERedDotType.java
package CommonEnum;

/*********
 * 红点类型
 **/
public enum ERedDotType {
	NONE, //0 ==== 无
	;
public static final ERedDotType[]  ERedDotType_Values = ERedDotType.values();
public static final int ERedDotType_Length = ERedDotType_Values.length;
public static ERedDotType ERedDotType_FromInt(int _ivalue) {
	if(_ivalue < 0 || _ivalue >= ERedDotType_Length){ return null; }
	return ERedDotType_Values[_ivalue];
}
}

File: game_client/Assets/Scripts/ClientProtocol/CommonEnum/ERedDotType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ALBasicProtocolPack;
using UnityEngine;


namespace CommonEnum
{

/// <summary>
/// 红点类型
/// </summary>
public enum ERedDotType {
	[InspectorName("NONE - idx[0] - 无")]
	NONE, //0 ==== 无
}

public class ERedDotTypeComparer : IEqualityComparer<ERedDotType>{
	public bool Equals(ERedDotType x, ERedDotType y) { return x == y; }
	public int GetHashCode(ERedDotType obj) { return obj.GetHashCode(); }
	public static int g_iEnumCount = 1;
}
}

